_oasis

```diff
@@ -52,3 +52,10 @@ Executable "biokepi-demo"
   CompiledObject: best
   BuildDepends: biokepi, threads
   MainIs:     main.ml
+
+Document biokepi
+  Title:                API reference for Biokepi
+  Type:                 ocamlbuild (0.4)
+  BuildTools:           ocamlbuild, ocamldoc
+  XOCamlbuildPath:      src/lib
+  XOCamlbuildLibraries: biokepi
```

Comment:
Why use Oasis?

It's missing the `-charset UTF-8 `, ` -keep-code` and `-colorize-code` options.

(And it's slower for 0 benefits).